### PR TITLE
Issue 1195 deployments page bulk actions

### PR DESCRIPTION
**Pull request by MTrost**

### CODE REVIEW

File: code/src/cljs/sixsq/nuvla/ui/deployments/utils.cljs [hunk 1]
```diff
@@ -160,20 +160,22 @@
...
 
 (defn build-bulk-filter
-  [{:keys [::spec/select-all?
-           ::spec/selected-set
-           ::spec/additional-filter
-           ::spec/state-selector] :as db}]
-  (if select-all?
-    (get-filter-param
+  ([{:keys [::spec/select-all?
+            ::spec/selected-set] :as db}]
```
Comment (0xbase12) [inline code/src/cljs/sixsq/nuvla/ui/deployments/utils.cljs:164]: Not working build-bulk-filter is returning empty string.
::spec/selected-set and ::spec/select-all? seems to be replaced by ::spec/select but not everywhere.
Events: ::events/select-all-page and ::events/select-all etc. not used. 

Something is not pushed?

Comment (MTrost) [reply code/src/cljs/sixsq/nuvla/ui/deployments/utils.cljs:164]: Forgot to remove the events, they are now part of the bulk selection feature.

Fixing other filter bug

Comment (MTrost) [reply code/src/cljs/sixsq/nuvla/ui/deployments/utils.cljs]: Should work now

**Review by 0xbase12 — APPROVED** [review]

File: code/src/cljs/sixsq/nuvla/ui/deployments/utils.cljs [hunk 2]
```diff
@@ -94,21 +95,35 @@
...
+  ([{:keys [::spec/additional-filter
+            ::spec/state-selector
+            ::spec/select] :as db}]
+   (table-plugin/build-bulk-filter
+     select
+     (get-filter-param
+       {:full-text-search  (full-text-search-plugin/filter-text
+                             db [::spec/deployments-search])
+        :additional-filter additional-filter
+        :state-selector    (when-not (= "all" state-selector) state-selector)
```
[the remark this answers is not in the document]
Comment (MTrost) [reply code/src/cljs/sixsq/nuvla/ui/deployments/utils.cljs:123]: Ah yes, that's why I changed the `state-filter` function! But then didn't finish the work by removing this check

**Pull request merged**